web/src/features/fba/components/infoPanel/AdvisoryText.tsx

```diff
@@ -41,6 +42,33 @@ const AdvisoryText = ({
     return 0
   }
 
+  // Return a list of fuel stats for which greater than 90% of the area of each fuel type has high HFI.
+  const getTopFuelsByProportion = (zoneUnitFuelStats: FireZoneFuelStats[]): FireZoneFuelStats[] => {
+    const topFuelsByProportion = zoneUnitFuelStats.filter(fuelStats => {
+      return fuelStats.area / fuelStats.fuel_area >= 0.9
+    })
+    return topFuelsByProportion
+  }
+
+  // Return a list of fuel types that cummulatively account for more than 75% of total area with high HFI.
+  const getTopFuelsByArea = (zoneUnitFuelStats: FireZoneFuelStats[]): FireZoneFuelStats[] => {
+    let totalHighHFIArea = 0
```

Comment:
Minor: Can be simplified with `reduce`:

`const totalHighHFIArea = zoneUnitFuelStats.reduce((total, stats) => total + stats.area, 0);`

Reply:
Changed in https://github.com/bcgov/wps/pull/4276/commits/2df6c9e330706e534583daa0b3a5884a38001889